pkg/kv/kvserver/replica_proposal.go

```diff
@@ -526,19 +526,15 @@ func addSSTablePreApply(
 	}
 
 	tBegin := timeutil.Now()
-	var tEndDelayed time.Time
 	defer func() {
 		if dur := timeutil.Since(tBegin); dur > addSSTPreApplyWarn.threshold && addSSTPreApplyWarn.ShouldLog() {
 			log.Infof(ctx,
-				"ingesting SST of size %s at index %d took %.2fs (%.2fs on which in PreIngestDelay)",
-				humanizeutil.IBytes(int64(len(sst.Data))), index, dur.Seconds(), tEndDelayed.Sub(tBegin).Seconds(),
+				"ingesting SST of size %s at index %d took %.2fs",
+				humanizeutil.IBytes(int64(len(sst.Data))), index, dur.Seconds(),
 			)
 		}
 	}()
 
-	eng.PreIngestDelay(ctx)
```

Comment:
Was this the only user of `PreIngestDelay` interface method? Consider removing that too.

Reply:
No, we continue to call it above raft, but note the TODO (which I put there):

https://github.com/cockroachdb/cockroach/blob/736a67e0d36cc545bf74d65db069ee895ff9bea0/pkg/kv/kvserver/store_send.go#L353-L354

@irfansharif do you know who would have strong opinions on this these days?

Reply:
We should be able to get rid if it above raft as well.

Reply:
Sounds good, sent https://github.com/cockroachdb/cockroach/pull/101530 for that.